.github/workflows/ci.yaml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container: oryd/sdk:v0.0.51
```

Comment:
We need to keep this in sync now as well?

Reply:
Yes, we do. But I do intend to clean up the Circle CI pipeline soon-ish as well.

And ideally, I'd like to see if we can split up all these SDK generation processes into different workflows (per language), that use their own base images. But not sure if there are many benefits to that yet.